android/library/src/main/java/com/liferay/mobile/screens/ddm/form/service/ISubmitService.kt

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ * <p>
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ * <p>
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.mobile.screens.ddm.form.service
+
+import com.liferay.apio.consumer.model.Thing
+import com.liferay.mobile.screens.ddl.model.Field
+
+/**
+ * @author Paulo Cruz
+ */
+interface ISubmitService {
```

Comment:
why interfaces?

Reply:
hmmm, I'm not sure, they are too Apio-driven (things, operations and so on) so it's not possible to implement using other structure. I agree that, in this case, we should remove the interface.